docs-2.0/2.quick-start/4.nebula-graph-crud.md

```diff
@@ -43,13 +43,13 @@ nebula> SHOW HOSTS;
 +-------------+-----------+-----------+--------------+----------------------+------------------------+
 ```
 
-From the **Status** column of the table in the return results, you can see that all the Storage services are online.
+From the **Status** column of the table in the returned results, you can see that all the Storage services are online.
 
-### Asynchronous implementation of creation and alteration
+### Create and alter in an async way
```

Comment:
With such a title, I would suppose that this subsection is going to tell me how to create and alter in an async way step by step. But it actually describes the fact that `CREATE` and `ALTER` statements are carried out in an async way.